cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/TemplateSetConfiguration.java

```diff
@@ -35,7 +45,26 @@ public TemplateSetConfiguration(List<String> groupIds, boolean allowSnapshots, L
     this.groupIds = groupIds;
     this.allowSnapshots = allowSnapshots;
     this.hideTemplates = hideTemplates;
-    this.mavenCoordinates = mavenCoordinates;
+    this.mavenCoordinates = checkandCovertToMavenCoordinates(mavenCoordinates);
+  }
+
+  private List<MavenCoordinate> checkandCovertToMavenCoordinates(List<String> mavenCoordinates) {
```

Comment:
Typo: checkAndConvertToMavenCoordinates
Javadoc missing too.
Would be better to split this method into 2 methods.
One for the check and one for the conversion.